src/pages/AddJob.jsx

```diff
@@ -49,134 +49,250 @@ export default function AddJob() {
+                return (
+                  <option value={type} key={index}>
+                    {type}
+                  </option>
+                );
+              })}
+            </select>
+          </div>
+
+          <div className="flex flex-row mb-4 w-1/2 items-center">
+            <label className="label flex basis-44 ml-4">Salary Range</label>
+            <div className="inline-flex w-full">
               <input
                 type="number"
-                className="input input-bordered"
-                {...register("minWage", { required: true })}
+                className="input input-bordered w-1/3"
+                placeholder="Min $"
+                {...register("wage.min", { required: true, valueAsNumber: true })}
               />
-              <span>USD</span>
-            </label>
-            <span> - </span>
-            <label className="input-group input-md">
-              <span>Max Wage</span>
+              <div>
+                <span className="ml-4 mr-4 text-center self-center text-4xl"> - </span>
+              </div>
+
               <input
                 type="number"
-                className="input input-bordered"
-                {...register("maxWage", { required: true })}
+                className="input input-bordered w-1/3"
+                placeholder="Max $"
+                {...register("wage.max", {
+                  required: true,
+                  valueAsNumber: true,
+                })}
               />
-              <span>USD</span>
-            </label>
+            </div>
           </div>
         </div>
-        <div className="form-control">
-          <label className="label" htmlFor="location">
-            Job Location
-          </label>
-          <input
-            type="text"
-            className="input input-bordered"
-            {...register("location", { required: true })}
-          />
+
+        <div className="flex flex-row items-center w-full mb-4">
+          <div className="flex flex-row items-center w-full">
+            <label className="label flex basis-44">Benefits</label>
+            <div className="flex flex-row w-full">
+              <div className="self-center items-center w-1/3">
+                <input
+                  type="checkbox"
+                  name="Medical"
+                  value={true}
+                  className="scale-125"
+                  {...register("benefit.hasMedical", {
+                    value: (v) => (v === "true" ? true : false),
+                  })}
+                />
+                <label className="pl-2">Medical</label>
+              </div>
+
+              <div className="self-center items-center w-1/3">
+                <input
+                  type="checkbox"
+                  name="other"
+                  value={true}
+                  className="scale-125"
+                  {...register("benefit.hasOthers", {
+                    value: (v) => (v === "true" ? true : false),
```

Comment:
if you just print `value` here. what would that be? Not sure but I guess it could be a boolean already?

Reply:
Sorry it was my mistake! I wrongly set the value and now it can appear as boolean.